src/book-demo.njk

```diff
@@ -8,12 +8,64 @@ meta:
     description: Book a demo to understand how FlowFuse helps you professionalize your Node-RED deployment.
 ---
```

Comment:
Now the pages are so alike, shouldn't it be a one layout and two pages that render them?

Reply:
Or alternatively, more partials?

Reply:
Done, one layout and two pages